Scenes/Enemies/Enemy1.gd

```diff
@@ -39,6 +39,10 @@ func _process(delta):
 				enemy_direction = Vector2(-1, 0)
 	self.global_position += enemy_direction * speed * delta
 	
+		# Tracks the position of this instance and destroys it after it gets out of sight
+	if self.global_position.x <= -100:
```

Comment:
This is a simple way of doing it. I'll accept this!

Keep in mind that Godot has also a VisibilityNotifier node that is better suited for this purpose!